File: works/1476.py
E, S, M = map(int, input().split())
cnt = 1
e = 1; s = 1; m = 1

while True:
    if (e == E) and (s == S) and (m == M): break
    cnt = cnt + 1

    e = e + 1 if e < 15 else 1
    s = s + 1 if s < 28 else 1
    m = m + 1 if m < 19 else 1

print(cnt)

File: works/tmp/1476.cs
using static System.Console;
using System;
using System.Collections;

namespace Project1
{
    public class Class1
    {
        public static void Main(string[] args)
        {
            int E, S, M;
            string[] s_tmp = ReadLine().Split();
            E = int.Parse(s_tmp[0]); S = int.Parse(s_tmp[1]); M = int.Parse(s_tmp[2]);

            int cnt = 1;
            int e = 1, s = 1, m = 1;
            while(true)
            {
                if (e == E && s == S && m == M) break;
                cnt++;

                if (e < 15) e++;
                else e = 1;

                if (s < 28) s++;
                else s = 1;

                if (m < 19) m++;
                else m = 1;
            }
            Write(cnt);

        }
    }
}
